### PR TITLE
postgresql: sync with latest cont-postgresql v1.0.0

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by praiskup** [comment]
This is against master, I'll be happy submit PRs against f21 & f22 once we are done with this.

**Comment by praiskup** [comment]
ping

**Comment by bkabrda** [comment]
LGTM, merging. Thanks for the contribution!

File: postgresql/root/usr/share/cont-docs/postgresql/HOME.txt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Note that this is for 1.0.0 API
+(MAJOR.MINOR.FIX).  Any change in MAJOR part of version introduces backward
+incompatibility (you should take into account that interchanging MAJOR container
+with MAJOR+N container will need some manual changes in your tooling).  On the
+other hand, incrementing MINOR version just adds some new features.
+
+
+Basic concepts
+--------------
+We try to push you to mount PostgreSQL datadir from host machine (see
```
Comment (cgwalters) [inline postgresql/root/usr/share/cont-docs/postgresql/HOME.txt]: Why?

Comment (praiskup) [inline postgresql/root/usr/share/cont-docs/postgresql/HOME.txt]: https://github.com/sclorg/rhscl2dockerfile/issues/23
Except for debugging, we haven't found any use-case for database container not having explicit
volume mount for data -- and it is much worse to start database server mistakenly without external data
directory.

**Pull request merged**